app/src/main/AndroidManifest.xml

```diff
@@ -26,11 +22,7 @@
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
-
-    <permission
```

Comment:
Saw this entry in stackoverflow. Hence removed it. https://stackoverflow.com/questions/14832911/android-map-v2-why-maps-receive-permission